src/Client/Constants/InterviewConstants.cs

```diff
@@ -5,6 +5,9 @@ public static class InterviewConstants
 {
     public static readonly int MaxQuestionCount = 10;
 
+    public static readonly string SetupDialogCategoriesParameter = "Categories";
+    public static readonly string SetupDialogTitle = "Interview Setup";
```

Comment:
There is a decent amount of strings already. We might start to use resources (resx). What do you think @thezerda ? 😊